users/faq-parts/usage.rst

```diff
@@ -87,6 +87,17 @@ block) from A, and then as A gets rescanned remove the files from A.
 A workaround would be to copy first from A to B, rescan B, wait for B to
 rebuild on remote ends, and then delete from A.
 
+Can I help initial sync by copying files manually?
+--------------------------------------------------
+
+If you have a large folder that you want to keep in sync over a not-so-fast network, and you have the possibility to move all files to the remote site in a faster manner, here is a procedure to follow:
```

Comment:
```suggestion
If you have a large folder that you want to keep in sync over a not-so-fast network, and you have the possibility to move all files to the remote device in a faster manner, here is a procedure to follow:
```

I'm sorry, I missed this one instance. Otherwise, everything looks fine for me.